http/racing_server.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { serve, ServerRequest } from "./server.ts";
+
+const addr = Deno.args[1] || "127.0.0.1:4501";
+const server = serve(addr);
+
+const body = new TextEncoder().encode("Hello 1\n");
+const body2 = new TextEncoder().encode("World 2\n");
```

Comment:
Try making the bodies larger than 64k (the max size of tcp packets).
```ts
const body = new Uint8Array(1024 * 1024); // 1MB
body.fill('a'.charCodeAt(0));
const body2 = new Uint8Array(1024 * 1024); // 1MB
body2.fill('b'.charCodeAt(0));
```

Reply:
Hmm this would be testing interleaving problem. My test included here would demonstrate reordering problem, which is a subset of the problem we have about pipelining. Should I still add this test?

Reply:
Added anyways